ObjectTracking/src/bin/basic_control_1d.rs

```diff
@@ -223,16 +359,19 @@ fn main() -> eyre::Result<()> {
                 features2d::DrawMatchesFlags::DEFAULT,
             )?;
             let position = disk.get(0)?.pt;
-            dbg!(&position);
+            // dbg!(&position);
             let z = alpha * position.y + (1.0 - alpha) * previous_z;
             // ser.write(str((z-200)*0.03))
-            println!("{}", (z - 200.0) * 0.03);
+            // println!("{}", (z - 200.0) * 0.03);
             previous_z = z;
         }
-        highgui::imshow(window, &disk_img)?;
-        if highgui::wait_key(10)? > 0 {
-            break;
-        }
+        // warn!("hi there");
+        warn!(disk_img = valuable(JpegImage::from_mat(&disk_img)?));
```

Comment:
This warn macro is where we encode the jpeg. If nobody is listening then it will happen once and then be skipped (if we wanted to make this production-ready then we would also make it skip the first time)